profile.d/00-pengwin.sh

```diff
@@ -1,5 +1,14 @@
-# enable external x display
-export DISPLAY=:0
+# check whether it is WSL1 for WSL2 by using gcc version kernel built with
+if[ $(grep -oE 'gcc version ([0-9]+)' /proc/version | awk '{print $3}') -gt 5 ]; then
```

Comment:
There is no chance that in the future gcc version be increased and still be in WSL1?

Reply:
It is still unsure. There is one way to check by phrasing `wsl -l -v`, so maybe I will try that